psi4/run_psi4.py

```diff
@@ -134,9 +135,9 @@
     if args["input"] == "input.dat":
         args["output"] = "output.dat"
     elif args["input"].endswith(".in"):
-        args["output"] = args["input"].replace(".in", ".out")
+        args["output"] = ".out".join(args["input"].rsplit(".in", 1))
     elif args["input"].endswith(".dat"):
-        args["output"] = args["input"].replace(".dat", ".out")
+        args["output"] = ".out".join(args["input"].rsplit(".dat", 1))
```

Comment:
Bikeshedding, I know, but I'd mildly prefer

`args["output"] = args["input"][:-3] + "out"`

or

```
inp = pathlib.Path(args["input"])
otp = inp.stem + '.out'
```

Reply:
The first one should work fine, but the second will have other consequences I think.

```python
p1 = Path("input.dat")
p2 = Path("/home/a/b/c/input.dat")
p3 = Path("./subdir1/input.dat")
p1.stem == p2.stem == p3.stem
# True
```
pathlib.Path.stem just returns the filename without extensions, So any directories will be lost. 
`psi4 -i "./calculation_a/name.in"` and `psi4 -i "./calculation_b/name.in"`  would overwrite each others outfiles.

Reply:
you're right
```
>>> from pathlib import Path
>>> aa = Path('mydir/snddir/asdfasdf.in.xyz.in')
>>> str(aa.parent / aa.stem) + '.out'
'mydir/snddir/asdfasdf.in.xyz.out'
```